packages/rs-dpp/src/data_contract/config/mod.rs

```diff
@@ -190,6 +220,25 @@ impl DataContractConfigSettersV0 for DataContractConfig {
     ) {
         match self {
             DataContractConfig::V0(v0) => v0.requires_identity_decryption_bounded_key = value,
+            DataContractConfig::V1(v1) => v1.requires_identity_decryption_bounded_key = value,
+        }
+    }
+}
+
+impl DataContractConfigGettersV1 for DataContractConfig {
+    fn granular_integer_types(&self) -> bool {
+        match self {
+            DataContractConfig::V0(_) => false,
+            DataContractConfig::V1(v1) => v1.granular_integer_types,
```

Comment:
sized_integer_types or fixed_width_integer_types might be better than granular_integer_types